Makefile

```diff
@@ -25,6 +52,16 @@ build-and-run-poseidon-consts-gen:
 	./zig-out/bin/poseidon_consts_gen
 	@zig fmt ./src/math/crypto/poseidon/gen/constants.zig
 
+build-compare-benchmarks: build_cairo_vm_cli build
```

Comment:
can you create a scripts directory and move this to a bash script? this would help with readability for the make file

Reply:
done